Tests/test_GenBank_unittest.py

```diff
@@ -688,6 +689,145 @@ def test_000_write_invalid_but_parsed_locus_line(self):
         self.assertEqual(out_lines[0], invalid_line)
 
 
+class GenBankScannerTests(unittest.TestCase):
+    """GenBank Scanner tests, test parsing gbk and embl files."""
+
+    gb_s = Scanner.GenBankScanner()
+
+    def gb_to_l_cds_f(self, filename, tags2id=None):
+        """Gb file to Seq list parse CDS features."""
+
+        handle = open(filename, 'r')
+        if tags2id:
+            l_cds_f = list(self.gb_s.parse_cds_features(handle, tags2id=tags2id))
+        else:
+            l_cds_f = list(self.gb_s.parse_cds_features(handle))
+        handle.close()
+        return l_cds_f
+
+    def gb_to_l_r(self, filename, do_features=False):
+        """Gb file to Seq list parse records."""
+
+        handle = open(filename, 'r')
+        if do_features:
+            l_gb_r = list(self.gb_s.parse_records(handle, do_features=True))
+        else:
+            l_gb_r = list(self.gb_s.parse_records(handle, do_features=False))
```

Comment:
Wouldn't one line instead of four be enough:

```python
l_gb_r = list(self.gb_s.parse_records(handle, do_features= do_features))
```

Reply:
yes, amended.